### PR TITLE
Service now create ticket, mirroring and state polling playbooks

### PR DESCRIPTION
**Pull request by EliorKedar**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description

ServiceNow playbook pack including:
- Create Ticket wrapper playbook
- Mirror Ticket playbook
- Ticket State Polling playbook
- Incoming & Outgoing mappers for ticket mirroring
- New ticket Mirroring layout for created tickets 
- General mirroring improvements 

## Screenshots
Create ticket
Create_ServiceNow_Ticket
Mirror Ticket 
Mirror_ServiceNow_Ticket
Ticekt State Polling
ServiceNow_Ticket_State_Polling

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ronykoz** [comment]
@yuvalbenshalom please assign this PR to whoever is helping @michalgold s team.

**Comment by yaron-libman** [comment]
Reviewed.

**Review by ronykoz — COMMENTED** [review]
Script looks good

**Review by michalgold — APPROVED** [review]

File: Packs/ServiceNow/Layouts/layoutscontainer-ServiceNow_Ticket.json [hunk 1]
```diff
@@ -413,7 +413,7 @@
    {
     "hidden": false,
     "id": "yyrfzsc55n",
-    "name": "ServiceNow Ticket",
+    "name": "ServieNow Ticket",
```
Comment (Itay4) [inline Packs/ServiceNow/Layouts/layoutscontainer-ServiceNow_Ticket.json:416]:
```suggestion
    "name": "ServiceNow Ticket",
```

File: Packs/ServiceNow/Playbooks/playbook-ServiceNow_Ticket_State_Polling_README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+| InstanceName | Set the ServiceNow Instance that will be used by the polling command.<br/>Only relevant when there is more than one ServiceNow instance. |  | Optional |
+| AdditionalPollingCommandName | Values of the additional arguments for the polling command, for example: \(value1,value2,...\). |  | Optional |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![ServiceNow Ticket State Polling](Insert the link to your image here)
```
Comment (Itay4) [inline Packs/ServiceNow/Playbooks/playbook-ServiceNow_Ticket_State_Polling_README.md:38]: need to add image here

File: Packs/ServiceNow/Playbooks/playbook-Create_ServiceNow_Ticket_README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+| MirrorCommentTags | Set tags for mirror comments and files to ServiceNow. | comments,work_notes,ForServiceNow | Optional |
+| FieldPolling | Set the value to true or false to determine if the paybook will execute the FieldPolling sub playbook.<br/>It is useful when it is needed to wait for the ServiceNow ticket to be resolved and continue the parent playbook.<br/>FieldPolling will run until the ticket state is either resolved or closed. | true | Optional |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![Create ServiceNow Ticket](Insert the link to your image here)
```
Comment (Itay4) [inline Packs/ServiceNow/Playbooks/playbook-Create_ServiceNow_Ticket_README.md:52]: need to add image here

File: Packs/ServiceNow/Playbooks/playbook-Mirror_ServiceNow_Ticket_README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+| FieldPollingInterval | Set interval time for the polling to run<br/>\(In minutes\) |  | Optional |
+| FieldPollingTimeout | <br/>Set the amount of time to poll the status of the ticket before declaring a timeout and resuming the playbook.<br/>\(In minutes\) |  | Optional |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![Mirror ServiceNow Ticket](Insert the link to your image here)
```
Comment (Itay4) [inline Packs/ServiceNow/Playbooks/playbook-Mirror_ServiceNow_Ticket_README.md:44]: need to add image here

**Pull request merged**